pallet/src/signer.rs

```diff
@@ -155,7 +161,7 @@ where
     fn default() -> Self {
         Multisig {
             signers: BoundedBTreeMap::<AccountId, SignerData<Balance>, Size>::new(),
-            block_height: BlockNumber::default(),
+            stored_block_height: None,
```

Comment:
Is this the background of wrapping the block_height in an `Option`? Because of the initial block_height of `0`? I think this cannot be mixed up, because in a substrate based blockchain, there is no block 0, it always starts with block 1... so you can still detect, that is hasn't been set...

Reply:
Nooo, I don't think so. The `None` is used for single signer scripts that don't care about the block info.
I can add a comment here for that